assets/findify-analytics.js

```diff
@@ -34,8 +34,8 @@
 
   const updateCartEvent = () => {};
 
-  setTimeout(() => {
+  window.addEventListener("findifyIsLoaded", () => {
     viewPageEvent();
     updateCartEvent();
-  }, 1000);
+  });
```

Comment:
Probably I already asked you and I forgot after 3 days of holiday.
Did you try the 
findify.core.init.then(() => { /* Do something here*/ })